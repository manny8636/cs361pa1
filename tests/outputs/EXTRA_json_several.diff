--- outputs/EXTRA_json_several.txt	2022-02-13 13:37:54.363624114 -0500
+++ expected/EXTRA_json_several.txt	2021-01-25 15:27:24.000000000 -0500
@@ -1 +1,9 @@
+Key-value pairs:
+{
+KEYS[a] = alpha
+KEYS[b] = beta
+KEYS[c] = gamma?
+KEYS[d] = 42
+KEYS[e] = 2.718281828459
+}
 Success!
