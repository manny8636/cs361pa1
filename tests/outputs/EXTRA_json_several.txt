Parsing ../data/json_several.txt failed
