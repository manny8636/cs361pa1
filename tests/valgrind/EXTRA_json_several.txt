==3062941== Memcheck, a memory error detector
==3062941== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3062941== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3062941== Command: ../json -o ../data/json_several.txt
==3062941== 
Success!
==3062941== 
==3062941== HEAP SUMMARY:
==3062941==     in use at exit: 89 bytes in 1 blocks
==3062941==   total heap usage: 4 allocs, 3 frees, 16,945 bytes allocated
==3062941== 
==3062941== LEAK SUMMARY:
==3062941==    definitely lost: 89 bytes in 1 blocks
==3062941==    indirectly lost: 0 bytes in 0 blocks
==3062941==      possibly lost: 0 bytes in 0 blocks
==3062941==    still reachable: 0 bytes in 0 blocks
==3062941==         suppressed: 0 bytes in 0 blocks
==3062941== Rerun with --leak-check=full to see details of leaked memory
==3062941== 
==3062941== For lists of detected and suppressed errors, rerun with: -s
==3062941== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
