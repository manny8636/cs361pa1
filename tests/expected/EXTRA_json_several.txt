Key-value pairs:
{
KEYS[a] = alpha
KEYS[b] = beta
KEYS[c] = gamma?
KEYS[d] = 42
KEYS[e] = 2.718281828459
}
Success!
